releasenotes/releasenotes_v5.rst

```diff
@@ -5,8 +5,129 @@ Current Firely Server release notes (v5.x)
+#. When executing a ``POST``-based search, Firely Server will now return the correct self-link as seen in ``GET``-based searches.
+#. The client id of the default SMART authorization options have been changed from ``vonk`` to ``firelyserver``.
+#. Firely Server now returns improved error messages if the client is not allowed to perform searches.
+#. Support for Firely Server using a SQLite database on arm64-based Macs was improved. 
+#. During SMART on FHIR v2 discovery, Firely Server now returns the ``grant_types_supported`` field.
+#. Firely Server now returns the correct CodeSystem ``http://terminology.hl7.org/CodeSystem/restful-security-service`` within the security section of its ``CapabilityStatement``. Before this change, the old R3 CodeSystem ``http://hl7.org/fhir/restful-security-service`` was falsely returned.
+#. Firely Server will now handle duplicate DLLs and assemblies more gracefully in case they were accidentally added to its plugin directory.
+#. When overwriting Search Parameters, the new Search Parameters will now be included in the CapabilityStatement instead of the overwritten ones. This feature was introduced with Firely Server ``4.7.0`` but broke in between the last releases.
+#. The two SearchParameters ``ConceptMap-target-uri`` and ``ConceptMap-source-uri`` for ``ConceptMap`` have been fixed.
+#. For FHIR STU3 and R4, ``Contract``, ``GuidanceResponse`` and ``Task`` have been added to the ``Patient`` compartment.
+#. Firely Server now returns a ``404`` and ``OperationOutcome`` when the status of a cancelled export is requested.
+#. When preloading resources via Firely Server's import feature, no more errors will be logged if subfolders are present.
+#. Warnings and errors with regards to ``AuditEvent`` indexing problems have been fixed and will no longer appear in the log.
+#. Searches on ``period`` elements that have equal start/end times either at the start or beginning of the year will now return the correct results. Previously, these searches did not return any results.
+#. The US Core ``patient`` search parameters have been fixed. They now only target ``Patient``, not ``Group`` and ``Patient``.
+#. The response for unsupported ``Prefer`` headers has been improved. The ``Prefer`` header's value is now included in the ``OperationOutcome``.
+#. Firely Server will now respond with a ``408`` instead of a ``500`` status code in case the ``$everything`` operation times out.
+#. Custom ``SearchParameters`` can now include the character '-' in ``code``.
+#. The copyright data in Firely Server's executable has been updated.
+#. Miscellaneous flaws in Firely Server's `Swagger documentation UI <_static/swagger>`_ have been fixed.
+
```

Comment:
I think [VONK-4079](https://firely.atlassian.net/browse/VONK-4079)
$everything time out - respond gracefully is missing

Reply:
It's included: 

```
#. Firely Server will now respond with a ``408`` instead of a ``500`` status code in case the ``$everything`` operation times out.
```

However, I added the word 'gracefully' explicitly: 

```
#. Firely Server will now respond more gracefully with a ``408`` instead of a ``500`` status code in case the ``$everything`` operation times out.
```